client/reader/start/card.jsx

```diff
@@ -1,32 +1,39 @@
 // External dependencies
 import React from 'react';
+import { connect } from 'react-redux';
+import debugModule from 'debug';
+import get from 'lodash/get';
 
 // Internal dependencies
 import Card from 'components/card';
-import SiteIcon from 'components/site-icon';
-import FollowButton from 'reader/follow-button';
 import StartPostPreview from './post-preview';
+import StartCardHeader from './card-header';
+import StartCardFooter from './card-footer';
+import { getRecommendationById } from 'state/reader/start/selectors';
 
-const StartCard = React.createClass( {
-	render() {
-		const site = null;
-		const showPostPreview = true;
-		return (
-			<Card className="reader-start-card">
-				<div className="reader-start-card__hero"></div>
-				<header>
-					<SiteIcon site={ site } size={ 70 } />
-					<h1>The Adventures of Casey</h1>
-					<p>Casey Schreiner shares his tips on planning your first solo camping trip around the world.</p>
-				</header>
-				{ showPostPreview ? <StartPostPreview /> : null }
-				<footer>
-					<div className="reader-start-card__follower-count">537,000 Followers</div>
-					<FollowButton siteUrl="http://jancavan.wordpress.com" />
-				</footer>
-			</Card>
-		);
-	}
-} );
+const debug = debugModule( 'calypso:reader:start' ); //eslint-disable-line no-unused-vars
+
+const StartCard = ( { recommendation } ) => {
+	const siteId = get( recommendation, 'recommended_site_ID' );
+	const postId = get( recommendation, 'recommended_post_ID' );
+	return (
+		<Card className="reader-start-card">
+			<div className="reader-start-card__hero"></div>
+			<StartCardHeader siteId={ siteId } />
+			{ postId ? <StartPostPreview siteId={ siteId } postId={ postId } /> : null }
+			<StartCardFooter siteId={ siteId } />
+		</Card>
+	);
+};
 
-export default StartCard;
+StartCard.propTypes = {
+	recommendationId: React.PropTypes.number.isRequired
+};
+
+export default connect(
+	( state, ownProps ) => {
+		return {
+			recommendation: getRecommendationById( state, ownProps.recommendationId )
```

Comment:
hmmm... we probably _could_ grab the site and post here and pass them as props down the chain... Since this is the list item, not the list itself, should be fine.